databases/week2/school-database.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE school
+    DEFAULT CHARACTER SET = 'utf8mb4';
```

Comment:
I am not able to see any task in your school database 🤔

Reply:
hm weird, added again, maybe I didn't save it as I remember doing it